### ISSUE TITLE
Login loop on ocis.ocis-keycloak.latest.owncloud.works

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/ocis#4818]
https://user-images.githubusercontent.com/214010/195785059-26afa855-10b0-4c78-b8a2-ff0fc0a7050a.mp4

Console + Network errors:

- [Console.txt](https://github.com/owncloud/web/files/9783335/Console.txt)
- [ocis.ocis-keycloak.latest.owncloud.works.har.zip](https://github.com/owncloud/web/files/9783341/ocis.ocis-keycloak.latest.owncloud.works.har.zip)

`Safari Version 16.0 (17614.1.25.9.10, 17614)`

**Comment by georglauterbach** [comment]
Similar issue here:

``` TXT
{"level":"error","error":"error connecting to nats cluster ocis-cluster: error connecting to nats at nats:9233 with tls enabled (true): x509: certificate is valid for localhost, not nats","time":"2022-10-14T16:15:05.185230278Z","caller":"github.com/cs3org/reva/v2@v2.10.1-0.20221012104058-ae7c58b9bffa/pkg/events/server/nats.go:65","message":"can't connect to nats (jetstream) server, retrying in 1m19.100549382s"}
```

**Comment by micbar** [comment]
should be fixed by #4867 

Waiting for the automated deployment.

**Comment by micbar** [comment]
fix confirmed.
